concordium-sdk/src/main/java/com/concordium/sdk/transactions/smartcontracts/parameters/AddressParam.java

```diff
@@ -0,0 +1,54 @@
+package com.concordium.sdk.transactions.smartcontracts.parameters;
+
+import com.concordium.sdk.transactions.InitName;
+import com.concordium.sdk.transactions.ReceiveName;
+import com.concordium.sdk.transactions.smartcontracts.Schema;
+import com.concordium.sdk.transactions.smartcontracts.SchemaParameter;
+import com.concordium.sdk.types.AbstractAddress;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Getter;
+
+import java.io.IOException;
+
+/**
+ * Wrapper class allowing {@link AbstractAddress} to be passed directly as a smart contract parameter.
+ */
+@JsonSerialize(using = AddressParam.AddressParamSerializer.class)
+@Getter
+public class AddressParam extends SchemaParameter {
```

Comment:
Explained in `AccountAddress` comment

Reply:
Same comment as above: 
1. The documentation can be made clearer I think.
2. Is it required by more than just that one example? Maybe it should be moved to the example then in order to avoid confusion.